apps/native/app/src/screens/family/family-overview.tsx

```diff
@@ -30,32 +46,34 @@ const { useNavigationOptions, getNavigationOptions } =
     },
   }))
 
-const FamilyMember = React.memo(({ item }: { item: any }) => {
-  const theme = useTheme()
-
-  return (
-    <View style={{ paddingHorizontal: 16 }}>
-      <TouchableHighlight
-        underlayColor={
-          theme.isDark ? theme.shades.dark.shade100 : theme.color.blue100
-        }
-        style={{ marginBottom: 16, borderRadius: 16 }}
-        onPress={() => {
-          navigateTo(`/family/${item.type}/${item.nationalId}`, {
-            id: item?.nationalId,
-          })
-        }}
-      >
-        <SafeAreaView>
-          <FamilyMemberCard
-            name={item?.name || item?.displayName}
-            nationalId={formatNationalId(item?.nationalId)}
-          />
-        </SafeAreaView>
-      </TouchableHighlight>
-    </View>
-  )
-})
+const FamilyMember = React.memo(
+  ({ item }: { item: ChildItem | SpouseItem }) => {
+    const theme = useTheme()
+
+    return (
+      <View style={{ paddingHorizontal: 16 }}>
+        <TouchableHighlight
+          underlayColor={
+            theme.isDark ? theme.shades.dark.shade100 : theme.color.blue100
+          }
+          style={{ marginBottom: 16, borderRadius: 16 }}
+          onPress={() => {
+            navigateTo(`/family/${item.type}/${item.nationalId}`, {
+              id: item?.nationalId,
+            })
+          }}
+        >
+          <SafeAreaView>
+            <FamilyMemberCard
+              name={item?.fullName ?? ''}
```

Comment:
No fallback to displayname?

Reply:
There is no displayName in the new endpoint.